apps/condo/domains/marketplace/schema/MarketCategory.test.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * Generated by `createschema marketplace.MarketCategory 'name:Text; parentCategory:Relationship:MarketCategory:CASCADE;'`
+ */
+
+const { makeLoggedInAdminClient, makeClient, expectValuesOfCommonFields, expectToThrowGQLError, expectToThrowInternalError } = require('@open-condo/keystone/test.utils')
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+} = require('@open-condo/keystone/test.utils')
+
+
+const { MARKET_CATEGORY_ERRORS } = require('@condo/domains/marketplace/constants')
+const { MarketCategory, createTestMarketCategory, updateTestMarketCategory } = require('@condo/domains/marketplace/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser, makeClientWithSupportUser } = require('@condo/domains/user/utils/testSchema')
+
+describe('MarketCategory', () => {
+    let admin, support, anonymous, user, checkReadAccessCategoryObj
+    beforeAll(async () => {
+        admin = await makeLoggedInAdminClient()
+        support = await makeClientWithSupportUser()
+        user = await makeClientWithNewRegisteredAndLoggedInUser()
+        anonymous = await makeClient();
```

Comment:
Doesn't `ESLint` swear at semicolons?

Reply:
the line following it [checkReadAccessCategoryObj] = await createTestMarketCategory(admin) this won't work if I remove the ; because this symbol tells js that the line is finished, otherwise it will try to merge the top and bottom line and an error will occur

Reply:
that's why he doesn't swear, because he sees an error if this rule is applied

Reply:
more precisely, this is not even the case that he processes. Without `;` the line would look like accessing an array element like `anonymous = await makeClient()[checkReadAccessCategoryObj] = await createTestMarketCategory(admin)`